ansible/configs/osp17-director-deployment/test.yml

```diff
@@ -10,7 +10,7 @@
       register: output_01_2_1
       loop:
         - ssh-keygen -b 2048 -t rsa -f ~/.ssh/id_rsa -q -N ""
-        - sshpass -p r3dh4t1! ssh-copy-id -o  StrictHostKeyChecking=no root@undercloud.example.com -p 22
+        - sshpass -p r3dh4t1! ssh-copy-id -o  StrictHostKeyChecking=no -p 22 root@undercloud.example.com
```

Comment:
SSH on port 22 with a known vulnerable password... yikes!

Also, this should be using `command` rather than `shell`.

Reply:
Sent this to @YogiSoni-lazy  to review